Cython/Utility/Coroutine.c

```diff
@@ -256,7 +256,7 @@ static PyObject *__Pyx_Coroutine_GetAsyncIter_Generic(PyObject *obj) {
     }
 #else
     // avoid C warning about 'unused function'
-    if ((0)) (void) __Pyx_PyObject_CallMethod0(obj, PYIDENT("__aiter__"));
+    (void) &__Pyx_PyObject_CallMethod0;
```

Comment:
I assume this doesn't work with `CYTHON_UNUSED_VAR(&__Pyx_PyObject_CallMethod0);`?

Reply:
That should work fine. I will update it.